apps/web/cypress/e2e/single-dwelling/single-dwelling-9.9.9-results.cy.ts

```diff
@@ -0,0 +1,47 @@
+import { TEMP_GET_URL_SINGLE_DWELLING_WALKTHROUGH } from "@repo/constants/src/urls";
+import {
+  EnumBuildingTypes,
+  EnumWalkthroughIds,
+} from "@repo/constants/src/constants";
+import {
+  getWalkthrough,
+  navigateThroughWalkthrough,
+} from "../../support/helpers";
+import { walkthroughs } from "../../fixtures/single-dwelling/single-dwelling-9.9.9-test-data.json";
+import {
+  GET_TESTID_RESULT_ITEM,
+  GET_TESTID_RESULT_PRINT_CONTENT_WALKTHROUGH,
+  GET_TESTID_RESULT_RELATED_ITEM,
+} from "@repo/constants/src/testids";
+
+describe("single dwelling: 9.9.9 walkthrough results", () => {
+  beforeEach(() => {
+    cy.visit(
+      TEMP_GET_URL_SINGLE_DWELLING_WALKTHROUGH(EnumWalkthroughIds._9_9_9),
+    );
+    navigateThroughWalkthrough(getWalkthrough(walkthroughs, 0));
+  });
+
+  it("should display correct ui elements", () => {
+    cy.getByTestID(GET_TESTID_RESULT_ITEM(EnumWalkthroughIds._9_9_9)).should(
+      "be.visible",
+    );
+    cy.getByTestID(
+      GET_TESTID_RESULT_RELATED_ITEM(EnumBuildingTypes.MULTI_DWELLING),
+    ).scrollIntoView();
+    cy.getByTestID(
+      GET_TESTID_RESULT_ITEM(EnumBuildingTypes.MULTI_DWELLING),
```

Comment:
@stechkev This needs to be `GET_TESTID_RESULT_RELATED_ITEM`. Copy paste error.

Reply:
```suggestion
      GET_TESTID_RESULT_RELATED_ITEM(EnumBuildingTypes.MULTI_DWELLING),
```